src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -156,14 +148,31 @@ struct EvolutionMetavars {
       dg::Events::Registrars::ObserveFields<1, Tags::Time, observe_fields,
                                             analytic_solution_fields>,
       Events::Registrars::ObserveTimeStep<EvolutionMetavars>,
-      Events::Registrars::ChangeSlabSize<slab_choosers>>>;
+      Events::Registrars::ChangeSlabSize>>;
 
   using observed_reduction_data_tags = observers::collect_reduction_data_tags<
       typename Event<events>::creatable_classes>;
 
   struct factory_creation
       : tt::ConformsTo<Options::protocols::FactoryCreation> {
     using factory_classes = tmpl::map<
+        tmpl::pair<
+            StepChooser<StepChooserUse::LtsStep>,
+            tmpl::conditional_t<
+                local_time_stepping,
+                tmpl::append<
+                    step_choosers_common<StepChooserUse::LtsStep>,
+                    step_choosers_for_step_only<StepChooserUse::LtsStep>>,
+                tmpl::list<>>>,
+        tmpl::pair<
+            StepChooser<StepChooserUse::Slab>,
+            tmpl::conditional_t<
+                local_time_stepping,
+                tmpl::append<step_choosers_common<StepChooserUse::Slab>,
+                             step_choosers_for_slab_only>,
+                tmpl::append<step_choosers_common<StepChooserUse::Slab>,
+                             step_choosers_for_step_only<StepChooserUse::Slab>,
+                             step_choosers_for_slab_only>>>,
```

Comment:
Perhaps define this whole configuration in a `Factory.hpp` so you don't have to repeat it in every executable, like the `StepControllers::standard_step_controllers`? Possibly templated on `bool LocalTimeStepping`

Reply:
I'll see what I can do.  Annoyingly, the list isn't the same in all the executables because some of the systems are missing implementations of functions needed for the CFL calculations.